Penteract/Assets/Scripts/SpawnPointController.h

```diff
@@ -56,6 +57,8 @@ class SpawnPointController : public Script {
 	GameObject* initialDoor = nullptr;
 	GameObject* finalDoor = nullptr;
 
+	GameObject* onCompletionActivateThis = nullptr;
```

Comment:
Please, rename this variable, it doesn't have meaning and it's not understandable